nbconvert/nbconvertapp.py

```diff
@@ -30,6 +31,24 @@
 #Classes and functions
 #-----------------------------------------------------------------------------
 
+def get_exporter(name):
+    """ given an exporter name, return a class ready to be instantiate
+    
+    Raises ValueError if exporter is not found
+    """
+    if name in exporter_map:
+        return exporter_map[name]
+
+    if '.' in name:
+        try:
+            return import_item(name)
+        except ImportError:
+            pass
```

Comment:
I'd like a little more detail on the error, perhaps:
```py
        try:
            return import_item(name)
        except ImportError as err:
            self.log.error("Error importing %s\n%s" % (name, err))
```

or combine it with the re-raise below....